### DISCUSSION TITLE
Genre getting assigned as URI, from string property.

### DISCUSSION THREAD

**Discussion by kirk-marple** [RehanSaeed/Schema.NET#45] [Q&A]
new MusicRecording { Genre = "indie" })

Where 'track' is MusicRecording, I'm finding that the genre is getting assigned as a System.Uri, not a string.  So this expression actually is null below.

track.Genre.Value.Value1.FirstOrDefault()

I'm guessing, since a Uri can be constructed from a string, it's mapping to that?  But 'indie' isn't a valid URI, and I can't figure out how to force it as a string.

**Comment by RehanSaeed** [comment]
This is fixed in the latest version. Thanks for reporting. Will release soon.

**Comment by kirk-marple** [comment]
Happy to test this out as soon as you have a moment to release it, thanks!

**Comment by kirk-marple** [comment]
Any chance of a release soon?  Would really like to try out your recent changes!

**Comment by RehanSaeed** [comment]
Need to write some more tests and get a higher level of confidence. Also trying to fix https://github.com/RehanSaeed/Schema.NET/issues/29
